src/store/storage.ts

```diff
@@ -26,13 +27,23 @@ function updateGroupType(context: ExtensionContext, groupType: GroupType) {
   commands.executeCommand("setContext", GROUP_TYPE_KEY, groupType);
 }
 
-export let storage: IStorage;
+export let followedUsersStorage: IStorage;
 export async function initializeStorage(context: ExtensionContext) {
-  storage = {
+  followedUsersStorage = {
     get followedUsers() {
+      tracing.appendLine(
```

Comment:
Should we retrieve the list of followed users (and sort them) once, and then log and return it? As opposed to doing it twice?

Also, we can probably update the log message to use prose as opposed to the internal state setting name (e.g. "followed users" vs. "followedUsers").